src/pages/library/dataBrowser-utils.ts

```diff
@@ -117,7 +116,7 @@ export const DatasetAccess = ({ dataset }: DatasetAccessProps) => {
       [access === datasetAccessTypes.Controlled, () => h(ButtonOutline, {
         style: buttonStyle,
         onClick: () => {
-          setRequestingAccess(true)
+          notify('info', 'Informal access request not yet supported through Terra, please contact the dataset owner')
```

Comment:
Is this better than disabling the button with a tooltip? I think in other places that's what we do, rather than a modal dialog with the button enabled.